app/assets/v2/css/profile.css

```diff
@@ -159,6 +163,8 @@
   max-height: 3.125rem;
   margin-right: auto;
   margin-left: auto;
+  border: 1px solid #E1E1E1;
```

Comment:
No border

Reply:
@PixelantDesign preferred the border because some icons looked weird without it.

Reply:
As far as I've seen we haven't used this color anywhere ! Could we take this out for now  ? 
I'll sync up offline with @PixelantDesign and add it in later with a different color (the green or yellow which is used right now on that page ) if needed

Reply:
Sounds good, let me remove those.